### PR TITLE
Docs/enrich architecture docs

### PR DESCRIPTION
**Pull request by ppxl**

### CODE REVIEW

File: docs/operations/figures/k8s-ces-overview-components_de.puml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+rectangle "Cluster" as cluster <<$k8s>> {
+  rectangle k8sapi as "K8s API-Server" <<$k8s>> #white
+  rectangle k8sres as "*"
+
+  rectangle "Namespace" as namespace <<$k8s>> #white {
+    rectangle mgrComps as "K8s-Komponenten" #white {
+
+      rectangle servdisc as "k8s-service-discovery" <<$cloudogu>>
+      note bottom of servdisc #ffffee
+        erzeugt Warp-Menü
```
Comment (nhinze23) [inline docs/operations/figures/k8s-ces-overview-components_de.puml:22]: The service discovery also creates the ingress resources and starts/stops the maintenance mode

File: docs/operations/figures/k8s-ces-overview-setup_de.puml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+actor admin
+
+'styling
+creds1 -[hidden]d- creds2
+
+admin --> setupres : installiert
+admin --> dogujson : installiert
+admin --> credentials : installiert
+dogujson -> setup
+servdisc -d-> cert : erzeugt (optional)
```
Comment (nhinze23) [inline docs/operations/figures/k8s-ces-overview-setup_de.puml:53]: The service-discovery always creates the secret. An own certificate has to be written to the etcd.

File: docs/operations/figures/k8s-ces-overview-setup_de.puml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+rectangle "Cluster" as cluster <<$k8s>> {
+  rectangle "Namespace" as namespace <<$k8s>> #white {
+
+    rectangle credentials as " " #white {
+      file cert as "<<secret>>\neigenes oder \nself signed \nZertifikat" MIXED_INSTALLATION
+      file creds1 as "<<secret>>\nDogu registry credentials" MANUAL_INSTALLATION
+      file creds2 as "<<secret>>\nImage registry credentials" MANUAL_INSTALLATION
+    }
+
+    file dogujson as "<<configmap>>\nk8s-dogu-json" MANUAL_INSTALLATION
```
Comment (nhinze23) [inline docs/operations/figures/k8s-ces-overview-setup_de.puml:23]: Do you mean setup.json? There is a setup-config with the operator versions etc. as well.

File: docs/operations/figures/k8s-dogu-operator-dogu-installation_de.puml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+dogurator <=== dogures : "erkennt Änderung \nim Cluster-Zustand"
+
+installMgr <--- dogujson   : "lädt dogu.json herunter (mit Credentials)"
+installMgr -> resourceGen  : "übergibt dogu.json"
+dogurator <-.-d- credentials  : "benutzt Credentials für\nZugang zu Registry-Diensten"
+deploymentres <-.-d- credentials  : "benutzt Credentials für\nZugang zu Registry-Diensten"
+resourceGen ====> doguResources : "legt Ressourcen anhand \nvon dogu.json an (upsert)"
+installMgr ----> k8setcd    : "schreibt dogu.json und \nggf. Konfigurationswerte in etcd"
+servdisc -> k8setcd        : "beobachtet auf Änderungen"
```
Comment (nhinze23) [inline docs/operations/figures/k8s-dogu-operator-dogu-installation_de.puml:63]: You could write here too that the servdisc watches the dogu resource services.

**Comment by nhinze23** [comment]
Nice work. These images make the understandig of the Multinode-CES much easier.

File: docs/operations/figures/k8s-ces-overview-components_de.puml (hunk 1 above)
Comment (ppxl) [reply docs/operations/figures/k8s-ces-overview-components_de.puml:22]: Ay, thanks, it seems there's a lot I forgot :+1:  Will add

- [ ] done

File: docs/operations/figures/k8s-ces-overview-setup_de.puml (hunk 3 above)
Comment (ppxl) [reply docs/operations/figures/k8s-ces-overview-setup_de.puml:23]: Hmm, this should be more refined, yes. 

- [ ] done

File: docs/operations/figures/k8s-dogu-operator-dogu-installation_de.puml (hunk 4 above)
Comment (ppxl) [reply docs/operations/figures/k8s-dogu-operator-dogu-installation_de.puml:63]: Good point. That is for disabling the "Dogu is Starting" Page, right?

- [ ] done

File: docs/operations/k8s-ces-architecture-overview_de.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+- [k8s-longhorn](https://github.com/cloudogu/k8s-longhorn)
+  - stellt eine Node-übergreifende Storage-Replikation allen Dogus mit Zustand zur Verfügung
+
+![PlantUML-Diagramm: Kubernetes-Cluster mit einer Dogu-Auswahl und wichtigen CES-Komponenten](figures/k8s-ces-overview-components_de.png "Kubernetes-Cluster mit einer Dogu-Auswahl und wichtigen CES-Komponenten")
+
+Umgekehrt heißt dies, dass diese Komponenten nicht im Single-Node Cloudogu EcoSystem eingesetzt werden können (vgl. Kapitel "Betriebssystempakete")
+
+## Setup-Prozess in einem Kubernetes Cluster
+
+Wegen der Eigenheiten von Kubernetes unterscheidet sich der Setup-Prozess von dem des Single-Node EcoSystem. Zusammenfassed wird sehr Ressourcen-orientiert gearbeitet und einige Dinge müssen von den Administrierenden manuell ausgebracht werden. Die Dokumentation von [`k8s-ces-setup` beschreibt](https://github.com/cloudogu/k8s-ces-setup/blob/develop/docs/operations/installation_guide_de.md) dies genauer.
```
[the remark this answers is not in the document]
Comment (nhinze23) [reply docs/operations/k8s-ces-architecture-overview_de.md:44]: ahh okay :+1:

File: docs/operations/k8s-ces-architecture-overview_en.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  - the Kubernetes-specific variant of `ces-setup` provides a basic installation of a Cloudogu EcoSystem using the components mentioned here
+- [k8s-dogu-operator](https://github.com/cloudogu/k8s-dogu-operator)
+  - manages Dogus in terms of their installation, updating, uninstallation and some more
+- [k8s-service-discovery](https://github.com/cloudogu/k8s-service-discovery)
+  - manages Dogus in terms of their accessibility in the Warp menu and the Cloudogu EcoSystem Registry
+  - manages TLS certificates of the Cloudogu EcoSystem
+- [k8s-component-operator](https://github.com/cloudogu/k8s-component-operator)
+  - manages K8s CES components in terms of their installation, update and uninstallation.
+- [k8s-etcd](https://github.com/cloudogu/k8s-etcd)
+  - provides the local Cloudogu EcoSystem registry
```
Comment (nhinze23) [inline docs/operations/k8s-ces-architecture-overview_en.md:30]:
```suggestion
  - provides the local Cloudogu EcoSystem Registry
```

File: docs/operations/k8s-ces-architecture-overview_en.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+In principle, all Dogus in a Cloudogu EcoSystem function in Kubernetes as they do in the single-node variant. However, the following exceptions exist.
+
+The dogu `official/nginx` is replaced by two dogus and one component in a Kubernetes environment: `k8s/nginx-ingress` and `k8s/nginx-static`. The reason for this is that several things are done simultaneously in Kubernetes that cannot be continued in this way in Kubernetes. In addition, there is the activity of an [ingress-controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/), which has no equivalent in a Docker-only world.
+
+The following table compares the functionalities and respective mapping components/dogus:
+
+| theme                   | K8s-CES               | conventional CES                                  |
+|-------------------------|-----------------------|---------------------------------------------------|
+| reverse proxy           | k8s/nginx-ingress     | official/nginx                                    |
```
Comment (nhinze23) [inline docs/operations/k8s-ces-architecture-overview_en.md:76]:
```suggestion
| Reverse proxy           | k8s/nginx-ingress     | official/nginx                                    |
```

**Comment by nhinze23** [comment]
lgtm

**Pull request merged**